### DISCUSSION TITLE
how can I change the default editor

### DISCUSSION THREAD

**Discussion by MohamedDhiaSleimi** [JaKooLit/Hyprland-Dots#361] [Q&A] [answered]
when I press the default key super+e to mess arround with the configs the config are opened in nano even thought i set up nvim as default in zsh I am somewhat new to linux so sorry if it is something obvious and thank you in advance

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/wiki/FAQ#%EF%B8%8F-dont-like-nano-or-kitty-for-quick-editor-or-super-e-quick-editor

**Reply by MohamedDhiaSleimi** [reply]
thank you allot
